scripts/rebalance-queue-masters

```diff
@@ -63,11 +63,12 @@ set_temp_queue_policies()
     local policy_set_master=''
     local -i count=0
     local -i master_idx=0
-    for queue in $(rabbitmqctl -q -p "$vhost" list_queues name | grep -E "$queue_regex")
+    IFS=$'\n'; for queue_and_pid in $(rabbitmqctl -q -p "$vhost" list_queues name pid| grep -E "$queue_regex")
```

Comment:
Great idea. I don't _think_ the semicolon is necessary before `for`. Also, could you pleas do something like this above:

```bash
local -r orig_IFS="$IFS"
```

and then restore `IFS` once the `for` loop is done. Thanks!